client/sites/overview/components/migration-overview/components/migration-ssh-complete.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { HostingCard } from 'calypso/components/hosting-card';
+import { HostingHeroButton } from 'calypso/components/hosting-hero';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { Container, Header } from './layout';
+import type { SiteDetails } from '@automattic/data-stores';
+
+export const MigrationSSHComplete = ( { site }: { site: SiteDetails } ) => {
+	const translate = useTranslate();
+	const sourceSiteDomain = site?.options?.migration_source_site_domain
```

Comment:
Another place to use `cleanUrl`?

Reply:
I created a new util function that can be share here.